adserver/admin.py

```diff
@@ -146,6 +148,7 @@ class AdvertisementAdmin(RemoveDeleteMixin, AdvertisementMixin, admin.ModelAdmin
 
     model = Advertisement
     save_as = True
+    list_per_page = 50  # make page load a bit faster
```

Comment:
This is fine for now. This admin would be *very* fast if we removed the aggregation across impressions. Maybe we should just do that.